### PR TITLE
Solana votes2

### PR DESCRIPTION
**Pull request by tarikceric**
Create Solana Votes2 model from streamline bronze data - [https://team-1612274056224.atlassian.net/browse/AN-2123](url)

- Filter all vote transactions from "bronze__transactions2"
- Reference logic from 'silver__transactions2" and original "silver__votes"
- Create yml file

Test results:
image

### CODE REVIEW

File: models/silver/silver__votes2.sql [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        t.data :meta :fee :: NUMBER AS fee,
+        CASE
+            WHEN IS_NULL_VALUE(
+                t.data :meta :err
+            ) THEN TRUE
+            ELSE FALSE
+        END AS succeeded,
+        t.data :transaction :message :accountKeys :: ARRAY AS account_keys,
+        t.data :transaction :message :instructions :: ARRAY AS instructions,
+        t.data :meta :innerInstructions :: ARRAY AS inner_instructions,
```
Comment (desmond-hui) [inline models/silver/silver__votes2.sql:24]: These can be removed, not used

File: models/silver/silver__votes2.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          - not_null: 
+              enabled: false # Vote authority not always available
+      - name: VOTE_HASH
+        description: Unique hash representing this set of votes(?)
+      - name: VOTE_SLOTS
+        description: Slots being voted on
+      - name: _INSERTED_TIMESTAMP
+        description: "{{ doc('_inserted_timestamp') }}"
+        tests: 
+          - not_null
```
Comment (desmond-hui) [inline models/silver/silver__votes2.yml:57]: `_partition_id` column should be part of the YML

**Review by desmond-hui — CHANGES_REQUESTED** [review]

File: models/silver/silver__votes2.sql (hunk 1 above)
Comment (tarikceric) [reply models/silver/silver__votes2.sql]: Instruction cols have been removed

File: models/silver/silver__votes2.yml (hunk 2 above)
Comment (tarikceric) [reply models/silver/silver__votes2.yml:59]: column has been added, but without test due to strange behavior

**Review by desmond-hui — APPROVED** [review]

**Pull request merged**